src/test/scala/viper/gobra/GobraOverflowCheckTests.scala

```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2011-2020 ETH Zurich.
+
+package viper.gobra
+
+import java.io.File
+
+import ch.qos.logback.classic.Level
+import viper.gobra.frontend.Config
+import viper.gobra.reporting.NoopReporter
+import viper.gobra.reporting.VerifierResult.{Failure, Success}
+import viper.silver.testing.{AbstractOutput, AnnotatedTestInput, ProjectInfo, SystemUnderTest}
+import viper.silver.utility.TimingUtils
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+sealed abstract class GobraOverflowCheckTests extends GobraTests {
```

Comment:
Would it be possible to replace this class by inline command line options?
I.e. move `resources/overflow_checks` to `resources/regressions/features/overflow_checks` and add the following line at the beginning of the files in the `int64` subdir:
```
// ##(--overflow --int64)
```
resp. for files in `int32` subdir:
```
// ##(--overflow)
```

Reply:
That is a great suggestion, I will change it!